capstone/src/main/java/com/google/sps/data/ReadData.java

```diff
@@ -4,52 +4,107 @@
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.sps.data.MapData;
+import com.google.sps.data.SentimentData;
 import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
 import java.io.FileReader;
 import java.io.IOException;
+import java.lang.NumberFormatException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.util.ArrayList;
 import java.util.Date;
+import java.util.Set;
+import java.util.stream.Stream;
 
 public class ReadData {
-  final String DELIMITER = ",";
+  private DatastoreService datastoreService;
+  private SentimentData sentimentService;
+  private ArrayList<Entity> newEntities;
+  private final String DELIMITER = ",";
+
+  public ReadData() throws IOException {
+    datastoreService = DatastoreServiceFactory.getDatastoreService();
+    sentimentService = new SentimentData();
+    newEntities = new ArrayList<Entity>();
+  }
+
+  /**
+  * Calls read file while iterating through all the zip codes
+  * @return {Void}
+  */
+  public void readAll() {
+    Set<String> zipCodes = MapData.zipPrecinctMap.keySet();
+    
+    for (String zip : zipCodes) {
+        String filePath = "assets/" + zip + ".csv";
+
+        try {
+            readFile(new FileReader(filePath), zip);
+        } catch (FileNotFoundException e) {
+            System.out.println(filePath + " is not in the assets folder");
+        }
+    }
+    
+    sentimentService.close();
+  }
 
   /**
-   * Reads the survey csv and adds each element to datastore
-   *
-   * @return {Void}
-   */
-  public void readCSV() {
+  * Reads the survey csv and adds each element to datastore
+  * @param file This is the file that is being parsed
+  * @param zip This is the zip code the file is associated with
+  * @return {Void}
+  */
+  public void readFile(FileReader file, String zip) {
+    final int EXPECTED_LENGTH = 15;
     BufferedReader reader = null;
-    FileReader file = null;
-    DatastoreService datastoreService = DatastoreServiceFactory.getDatastoreService();
 
     try {
-      file = new FileReader("assets/surveyresponse.csv");
       reader = new BufferedReader(file);
       String header = reader.readLine();
       String line = "";
-
+    
       while ((line = reader.readLine()) != null) {
         String[] values = line.split(DELIMITER);
-        String id = values[0] + "-" + values[1];
+        String id = values[0];
```

Comment:
? I thought what you had before was good?

Reply:
The arrangement of the csv changed, Survey ID is no longer a column in the csv

Reply:
I am not sure how to get the survey ID, for the MVP and for this PR the csvs were manually downloaded and titled based on their zip code i.e. 94116.csv